net/alertmanager/Portfile

```diff
@@ -0,0 +1,130 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        prometheus alertmanager 0.17.0 v
+github.tarball_from archive
+
+description         The Alertmanager handles alerts sent by client \
+                    applications such as the Prometheus server.
+
+long_description    AlertManager takes care of deduplicating, grouping, and \
+                    routing them to the correct receiver integration such as \
+                    email, PagerDuty, or OpsGenie. It also takes care of \
+                    silencing and inhibition of alerts.
+
+homepage            https://prometheus.io/docs/alerting/alertmanager/
+
+platforms           darwin
+categories          net
+license             apache
```

Comment:
please as specific as possible with the license, here I think it should be ```Apache-2``` (see [here](https://github.com/prometheus/alertmanager/blob/master/LICENSE))

Reply:
Fixed